src/main/java/edu/harvard/dbmi/avillach/dictionary/dashboarddrawer/DashboardDrawerRowMapper.java

```diff
@@ -0,0 +1,35 @@
+package edu.harvard.dbmi.avillach.dictionary.dashboarddrawer;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Array; // For handling SQL Array
+import java.util.Arrays;
+import java.util.List;
+
+public class DashboardDrawerRowMapper implements RowMapper<DashboardDrawer> {
+
+    @Override
+    public DashboardDrawer mapRow(ResultSet rs, int rowNum) throws SQLException {
+        return new DashboardDrawer(
+            rs.getInt("dataset_id"), rs.getString("study_fullname"), rs.getString("study_abbreviation"),
+            convertSqlArrayToList(rs.getArray("consent_groups")), rs.getString("study_summary"),
+            convertSqlArrayToList(rs.getArray("study_focus")), rs.getString("study_design"), rs.getString("sponsor")
+        );
+    }
+
+    private List<String> convertSqlArrayToList(Array sqlArray) throws SQLException {
+        if (sqlArray == null) {
+            return List.of();
+        } else {
+            Object[] arrayContents = (Object[]) sqlArray.getArray();
+            // Check if the array contains a single empty value
+            if (arrayContents.length == 1 && "".equals(arrayContents[0])) {
```

Comment:
This is a bit hacky but sqlArray returns an single empty value for empty values in the db.  Which makes sense as empty values can have meaning.  For this component they do not.